language/tools/move-cli/README.md

```diff
@@ -128,7 +128,7 @@ directory:
 Std = "0x1" # Specify and assign 0x1 to the named address "Std"
 
 [dependencies]
-MoveNursery = { git = "https://github.com/diem/move.git", subdir = "language/move-stdlib/nursery", rev = "d45f20a" }
+MoveNursery = { git = "https://github.com/move-language/move.git", subdir = "language/move-stdlib/nursery", rev = "d45f20a" }
 #                ^                    ^                     ^                                       ^
 #            Git dependency       Git clone URL       Subdir under git repo (optional)           Git revision to use
```

Comment:
These misaligned..

Reply:
Nice catch! I'll make sure to fix it in the next PR.